### PR TITLE
[MIG] 9.0 intrastat_base

### PR DESCRIPTION
**Pull request by thomaspaulb**
Migrated and added tests.

### CODE REVIEW

File: intrastat_base/data/country_data.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<odoo noupdate="1">
+<openerp>
+<data noupdate="1">
```
Comment (hbrunn) [inline intrastat_base/data/country_data.xml]: better leave `<odoo>`, that's the new standard in 9 too

File: intrastat_base/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (hbrunn) [inline intrastat_base/tests/__init__.py]: use short headers, and update your licence template ;-)

File: intrastat_base/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+from . import intrastat_base_tests
+
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (hbrunn) [inline intrastat_base/tests/__init__.py]: modelines are scorned upon by now

File: intrastat_base/tests/intrastat_base_tests.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```
Comment (hbrunn) [inline intrastat_base/tests/intrastat_base_tests.py:1]: this is valuable code too and deserves your license

File: intrastat_base/tests/intrastat_base_tests.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    def test_20_company(self):
+        # add 'Demo user' to intrastat_remind_user_ids
+        demo_user = self.env.ref('base.user_demo')
+        demo_company = self.env.ref('base.main_company')
+        demo_company.write({
+            'intrastat_remind_user_ids': [(6, False, [demo_user.id])]
+        })
+        # then check if intrastat_email_list contains the email of the user
+        self.assertEquals(demo_company.intrastat_email_list, demo_user.email)
+
```
Comment (hbrunn) [inline intrastat_base/tests/intrastat_base_tests.py]: remove empty lines

**Review by hbrunn — CHANGES_REQUESTED** [review]
only a few nags

File: intrastat_base/tests/intrastat_base_tests.py [hunk 6]
```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+# © 2016 Sunflower IT (http://sunflowerweb.nl).
```
Comment (thomaspaulb) [inline intrastat_base/tests/intrastat_base_tests.py:2]: This file was purely added by me, but I don't know what is the policy for that?

Comment (hbrunn) [reply intrastat_base/tests/intrastat_base_tests.py:2]: the policy is if you add something, it's your copyright

**Review by hbrunn — APPROVED** [review]

**Review by alexis-via — APPROVED** [review]

**Pull request merged**